packages/lib-classifier/package.json

```diff
@@ -50,18 +51,11 @@
     "styled-components": "~4.1.x"
   },
   "devDependencies": {
-    "@babel/core": "~7.7.2",
-    "@babel/plugin-proposal-class-properties": "~7.7.0",
-    "@babel/plugin-proposal-decorators": "~7.7.0",
-    "@babel/plugin-proposal-object-rest-spread": "~7.6.2",
-    "@babel/plugin-proposal-optional-chaining": "~7.8.3",
-    "@babel/polyfill": "~7.7.0",
-    "@babel/preset-env": "~7.7.1",
-    "@babel/preset-react": "~7.7.0",
-    "@babel/register": "~7.7.0",
     "@babel/core": "~7.8.4",
     "@babel/plugin-proposal-class-properties": "~7.8.3",
     "@babel/plugin-proposal-decorators": "~7.8.3",
+    "@babel/plugin-proposal-object-rest-spread": "~7.6.2",
+    "@babel/plugin-proposal-optional-chaining": "~7.8.3",
```

Comment:
I appear to have made a mistake in my rebase when updating dependencies earlier in the week. Could you submit a fix for this in a separate PR?

Reply:
I opened https://github.com/zooniverse/front-end-monorepo/pull/1497 to fix this.